code/modules/mining/equipment/wormhole_jaunter.dm

```diff
@@ -36,18 +37,25 @@
 
 	return destinations
 
-/obj/item/wormhole_jaunter/proc/activate(mob/user, adjacent)
+/obj/item/wormhole_jaunter/proc/activate(mob/user, adjacent, teleport)
 	if(!turf_check(user))
 		return
 
-	var/list/L = get_destinations(user)
+	var/list/L = get_destinations()
```

Comment:
Is this in the same proc chain as `chasm_react()`?

Reply:
`get_destinations` is a jaunter proc.  The user arg was never used so I removed it.